.lando.yml

```diff
@@ -35,6 +43,12 @@ services:
     type: mailhog
     hogfrom:
       - appserver
+  node:
+    type: node:10
+    build:
+      # Uncomment preferable tool for installing dependencies 
```

Comment:
Here too, we should only have one tool by default.

Also, since we provide a package.json in the repository, we don't need to comment this out. This will be different for existing projects, but we should remember that this repository is just a starting point.